apps/server/src/modules/lesson/controller/dto/lesson-resources.model.ts

```diff
@@ -0,0 +1,26 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
+
+export class LessonResources {
+	@ApiProperty({ description: 'client' })
+	client: string;
+
+	@ApiProperty({ description: 'description' })
+	description: string;
+
+	@ApiPropertyOptional({ description: 'merlinReference' })
+	merlinReference?: string;
+
+	@ApiProperty({ description: 'title' })
+	title: string;
+
+	@ApiPropertyOptional({ description: 'url' })
+	url?: string;
```

Comment:
Please add the public modifier here so that there are no more linter warnings.

Reply:
done

Reply:
Done